wagtail/admin/templates/wagtailadmin/home/pages_for_moderation.html

```diff
@@ -39,8 +44,10 @@ <h2>
                                             <input type="submit" class="button button-small button-secondary no" value="{% trans 'Reject' %}">
                                         </form>
                                     </li>
-                                    <li><a href="{% url 'wagtailadmin_pages:edit' revision.page.id %}" class="button button-small button-secondary">{% trans 'Edit' %}</a></li>
-                                    <li><a href="{% url 'wagtailadmin_pages:preview_for_moderation' revision.id %}" class="button button-small button-secondary" target="_blank" rel="noopener noreferrer">{% trans 'Preview' %}</a></li>
+                                    {% if page_perms.can_edit %}
+                                        <li><a href="{% url 'wagtailadmin_pages:edit' revision.page.id %}" class="button button-small button-secondary">{% trans 'Edit' %}</a></li>
+                                    {% endif %}
+                                    <li><a href="{% url 'wagtailadmin_pages:preview_for_moderation' revision.id %}" class="button button-small button-secondary" target="_blank">{% trans 'Preview' %}</a></li>
```

Comment:
You missed `rel="noopener noreferrer"` while resolving conflicts. It was added recently in 303ee0f95b0b1911826a4ef8f4aeda72ad00a4cb (see [this](https://github.com/wagtail/wagtail/commit/303ee0f95b0b1911826a4ef8f4aeda72ad00a4cb#diff-1dda5b756ea9f7b9df0481e35eeb9fb2R43)).